bot/exts/filters/filter_lists.py

```diff
@@ -55,17 +55,20 @@ async def _add_data(
         """Add an item to a filterlist."""
         allow_type = "whitelist" if allowed else "blacklist"
 
-        # If this is a server invite, we gotta validate it.
+        # If this is a guild invite, we gotta validate it.
         if list_type == "GUILD_INVITE":
             guild_data = await self._validate_guild_invite(ctx, content)
             content = guild_data.get("id")
 
-            # Unless the user has specified another comment, let's
-            # use the server name as the comment so that the list
-            # of guild IDs will be more easily readable when we
-            # display it.
-            if not comment:
-                comment = guild_data.get("name")
+            # Some guild invites are autoban filters, which require the mod
+            # to set a comment which includes [autoban].
+            # Having the guild name in the comment is still useful when reviewing
+            # filter list, so prepend it to the set comment in case some mod forgets.
+            comment = " - ".join(
+                comment_part
+                for comment_part in (guild_data.get("name", ""), comment)
```

Comment:
A format like `Guild "name here" - comment here` would be even clearer. It's not self-evident what the part before the dash means. For example, a guild named "123"

![bild](https://user-images.githubusercontent.com/1515135/174496765-8be1010d-bab3-4ac4-a0e9-23567020f89c.png)

Quoting the guild name will also help set the boundary between the comment and the guild name. Otherwise, a guild name with a dash in it could lead to further confusion.

Reply:
https://github.com/python-discord/bot/pull/2196/commits/af4339e1436bf554188205341a7b82becb573af1